app/src/main/java/org/sopt/dateroad/presentation/ui/component/chip/DateRoadAreaChip.kt

```diff
@@ -0,0 +1,61 @@
+package org.sopt.dateroad.presentation.ui.component.chip
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.sopt.dateroad.R
+import org.sopt.dateroad.ui.theme.DateRoadTheme
+import org.sopt.dateroad.ui.theme.DeepPurple
+import org.sopt.dateroad.ui.theme.Gray100
+import org.sopt.dateroad.ui.theme.Gray400
+import org.sopt.dateroad.ui.theme.White
+
+@Composable
+fun DateRoadAreaChip(
+    textId: Int,
```

Comment:
```suggestion
    @StringRes textId: Int,
```
String 파일에서 불러오는 값들은 앞에 StringRes 어노테이션을 추가해 주세요!